.github/workflows/test_rust.yml

```diff
@@ -76,6 +76,10 @@ jobs:
         # Don't fail the build for clippy on nightly, since we get a lot of false-positives
         run: cargo clippy --all --all-features --tests ${{ (matrix.rust_version != 'nightly' && '-- -D warnings') || '' }}
 
+      - name: cargo-deny
```

Comment:
I'd maybe limit this to *one* of the Linuxes

Reply:
I've limited it to Linux and Rust stable.